src/main/java/org/mskcc/cbio/oncokb/service/impl/CompanyServiceImpl.java

```diff
@@ -168,8 +181,30 @@ public Optional<CompanyDTO> findOneByNameIgnoreCase(String name) {
     }
 
     @Override
+    @Transactional()
     public void delete(Long id) {
         log.debug("Request to delete Company : {}", id);
+        Optional<CompanyDTO> maybeCompanyDTO = this.findOne(id);
+        if (!maybeCompanyDTO.isPresent()) {
+            return;
```

Comment:
Could you check what's the current behavior when id is not available? Do we throw an exception?